app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -352,6 +352,7 @@ class ChatRoomPresenter @Inject constructor(
                                 RoomUiModel(roles = chatRoles, isBroadcast = chatIsBroadcast)
                             ), false
                         )
+                        view.clearMessageComposition(true)
```

Comment:
Doing that, if something went wrong when calling the ` client.sendMessage(...)` will cause the users lost their messages, needing it to be written again.

Reply:
But the unsent message will still be displayed (greyed out) so the user won't lose the message. Am I wrong?